CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## 2.32.0-rc.0 / 2021-12-01
+
+This list of changes is relative to [v2.32.0-beta.0](https://github.com/prometheus/prometheus/releases/tag/v2.32.0-beta.0), which was a special 2.32 beta release to pre-publish the Prometheus Agent features.
+
+* [FEATURE] TSDB: Add support for forwarding exemplars in Agent mode. #9664
+* [FEATURE] UI: Adapt web UI for Prometheus Agent mode. #9851
+* [ENHANCEMENT] Linode SD: Tune API request page sizes. #9779
+* [ENHANCEMENT] TSDB: Check postings bytes length when decoding. #9766
+* [BUGFIX] TSDB: Fix panic when checkpoint directory is empty. #9687
+* [BUGFIX] Web: Fix a potential goroutine leak by channel. #8965
+* [BUGFIX] TSDB: Fix panic, out of order chunks, and race warning during WAL replay. #9856
+* [BUGFIX] TSDB: Fix and improve the `Seek()` method of various iterators. #9878
```

Comment:
This is not user facing (I think)

Reply:
This one does sound minorly user-facing to me, at least when reading the description in https://github.com/prometheus/prometheus/pull/9878

At least it sounds like there can be a performance impact due to the bug fixes, but potentially a user-visible correctness fix as well, /cc @beorn7 for comment.

Reply:
I would assume this bug had no effect so far. (I only found it because I was using those iterators in new ways for the new histograms.) Most likely, the `Seek` method wasn't used on these iterators so far, or at least not repeatedly.

Since nobody complained about any wrong behavior, we could go with that assumption and not mention this change.

Reply:
Thanks for the input! Ok, I removed this as well now.